src/main/java/com/google/cloud/teleport/splunk/SplunkEventWriter.java

```diff
@@ -317,6 +318,9 @@ abstract Builder setDisableCertificateValidation(
      */
     public Builder withUrl(ValueProvider<String> url) {
       checkArgument(url != null, "withURL(url) called with null input.");
+      if (url.isAccessible()) {
+        checkArgument(url.get().startsWith("http"), "input url has no protocol.");
```

Comment:
This check doesn't fully address the isssue.  If input is provides as ```httpbadurl``` then input passes the check but still throws a MalformedURLException.

Instead can you verify the simple pattern like ```"^http(s?):\\/\\/.+$"```  to ensure Url matches either http://domain or https://domain

Reply:
Done